package.json

```diff
@@ -55,7 +58,8 @@
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
     "react-ga": "^2.5.6",
-    "react-styleguidist": "^8.0.6",
+    "react-styleguidist": "9.0.0",
```

Comment:
We'll want to include AC about our MDS site in general since we're upgrading Styleguidist a major version.

Reply:
Also, unless we have a specific reason for doing so: we want to allow for any MINOR and PATCH upgrades to our dependencies.

Something like: `^9` or `^9.0.0`

Reply:
@juanca where should that new AC go, here in the PR, in the pivotal issue, or both?

Reply:
Both! We want it here to lead by example (as an open source library). We want it in pivotal for our QA fellows.

Reply:
Excellent, that's what has been done. :D